template/src/ontology/Makefile.jinja2

```diff
@@ -342,6 +341,26 @@ $(REPORTDIR)/$(SRC)-obo-report.tsv: $(SRCMERGED) | $(REPORTDIR)
 
 $(REPORTDIR)/%-obo-report.tsv: % | $(REPORTDIR)
 	$(ROBOT) report -i $< $(REPORT_LABEL) $(REPORT_PROFILE_OPTS) --fail-on $(REPORT_FAIL_ON) {% if project.robot_report.use_base_iris %}{% if project.namespaces is not none %}{% for iri in project.namespaces %}--base-iri {{ iri }} {% endfor %}{% else %}--base-iri $(URIBASE)/{{ project.id.upper() }}_ --base-iri $(URIBASE)/{{ project.id }} {% endif %}{% endif -%} --print 5 -o $@
+{%- if project.robot_report.upper_ontology is defined and project.robot_report.upper_ontology %}
+
+$(REPORTDIR)/$(SRC)-align-report.tsv: $(SRCMERGED) | $(REPORTDIR) all_robot_plugins
+	$(ROBOT) odk:validate -i $< --reasoner $(REASONER) \
+		 --upper-ontology-iri {{ project.robot_report.upper_ontology }}
+		 {%- if project.robot_report.use_base_iris %} \
+		 {% if project.namespaces is not none %}{% for iri in project.namespaces %}--base-iri {{ iri }} \
+		 {% endfor %}{% else %}--base-iri $(URIBASE)/{{ project.id.upper() }}_ --base-iri $(URIBASE)/{{ project.id }} \
+		 {% endif %}{% else %} \
+		 {% endif %}--report-output $@
```

Comment:
I have some additional thoughts here (what would be useful in that report), not sure if you want to discuss this here?

Reply:
Why not in the dedicated issue (#1175)?